### ISSUE TITLE
Add full customization of every possible presence that exist currently

### ISSUE THREAD

**Issue by TheJoshue** [hmes98318/Music-Disc#57]
**Describe the Issue/Feature**
Implement all available presence types, add new option:
```yml
BOT_ACTIVITY_TYPE
```
which can be: `PLAYING`, `LISTENING`, `STREAMING`, `CUSTOM`, where if set to CUSTOM, it will not be prefixed with "playing... <whatever>"
Also i would advise (this is just a suggestion!) just to match this env variable, changing:
```yml
# from
BOT_PLAYING
# to
BOT_ACTIVITY_STATUS
```

**Why this issue to happened / feature is needed(give scenarios or use cases):**
This feature would cover everything related to presence activity

**Screenshots**
/

**Version Information:**
/

**Additional context**
/

**Comment by hmes98318** [comment]
Yes, modifying different ActivityTypes is a good feature and can be added in future version.  
https://discord.com/developers/docs/topics/gateway-events#activity-object-activity-types  


This needs to modify the setActivity function of the ready event.  
https://github.com/hmes98318/Music-Disc/blob/138f3bec3e592f48bb37c482b7e47791c671a807/src/events/discord/ready.ts#L43
